packages/client/src/redux/slices/postSlice.ts

```diff
@@ -31,15 +32,19 @@ export enum PostSortType {
 type PostState = {
   posts: Post[];
   sortType: PostSortType;
-  locationFilter: string;
+  locationFilter: Location;
```

Comment:
we can prob get rid of this prop and related actions in this 🍕 since we have a location 🍕  now

Reply:
I thought we still needed to keep track of location of posts in order to filter them

Reply:
o `locationFilter` was meant to represent the current location the user has currently selected, which is basically ur slice